docs/docs/software-architecture.md

```diff
@@ -31,139 +31,147 @@ UI - User Interface
 
+   
 ## 5. Architectural Goals and Constraints
 
-There are some key requirements and system constraints that have a significant bearing on the architecture. They are:ƒ
+There are some key requirements and system constraints that have a significant bearing on the architecture. They are:
 
 - The Swapchain platform should satisfy all requirements set by the AMOS project, i.e. licenses, schedule, tools, etc.
 - The BOM should be within a reasonable amount.
 - Swapchain platform should be accessible with local and remote PCs.
 - All private and public information of the clients using the platform should be secured safely.
-- The Swapchain platform will be implemented as a client-server system. The client portion resides on PCs and the server portion must operate on the APIs and databases used.
+- The Swapchain platform will be implemented as a client-server system. The client portion resides on PCs and the server portion must operate on the APIs used.
 
+   
 ## 6. Deployment
 
-This UI is a system which is hosted in an API. The database will be hosted by ChainSquad once the software is delivered. Atomic transactions are conducted in the backend, so that the client computer does not have to spend much of CPU power.
-
+This UI is an API which is hosted in Netlify. After finishing the AMOS course and therewith the project, the software will be hosted by ChainSquad. Atomic transactions are conducted in the backend, so that the client computer does not have to spend much of CPU power.
+   
 ### 6.1. Technology Stack Description
 
 All services will be running in docker containers at first and orchestrated by docker-compose for simplicity sake. [1][2]
 
-Frontend:
+Frontend:   
 
 - JavaScript, CSS, HTML
 - React.tsx
+- Tailwind.css
 - Nginx container
 
-Backend:
+Backend:   
 
 - Microservice architecture
 - TypeScript, Node.js, Express
 - Communication is JSON over HTTP
-- Docker / Kubernetes
+- Docker
 
-Blockchain Gateway:
+Blockchain Gateway:   
 
 - Bitcoin libraries:
-  - Bitcoinjs (junderw, 2020)
+  - Bitcoinjs-lib (junderw, 2020)
 - Bitshare libraries:
-  - Bitsharejs
+  - bitsharesjs
 
-HTLC/XXX:
+HTLC:   
 
 - Creates the HTLC on its own blockchain
 
-Database:
-
-- Mongoose/MongoDB
-
-Logging:
+Logging:   
```

Comment:
Can be removed, we aren't logging anything, this would go against he privacy constraints in most cases ;)